tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_homescreen_change_wallpaper.py

```diff
@@ -31,9 +30,9 @@ def test_homescreen_change_wallpaper_from_gallery(self):
         self.take_screenshot()
 
         activities = contextmenu.tap_change_wallpaper()
-        self.take_screenshot()
-
-        # select gallery
+        # it is currently in system frame, but self.apps.switch_to_displayed_app() will return
+        # a different value
```

Comment:
Which value does it return then? The homescreen?
It's strange that it does return something else, because this context menu is part of the system app and it's the only thing that's visible, so I would think that self.apps.switch_to_displayed_app() does return the system app here.